subprojects/ide/build.gradle.kts

```diff
@@ -47,6 +47,7 @@ dependencies {
     testImplementation(libs.equalsverifier)
     testImplementation(testFixtures(project(":core")))
     testImplementation(testFixtures(project(":dependency-management")))
+    testImplementation(testFixtures(project(":language-groovy")))
```

Comment:
Why do we need this now? Would it make sense to add a `because` here?

Reply:
It's because we now refer to `GroovyCoverage` which is in `:language-groovy`. I don't know if adding a `because` is helpful, though.